app/controllers/registrations_controller.rb

```diff
@@ -307,10 +307,12 @@ def stripe_webhook
       logger.warn "Stripe webhook error while parsing basic request. #{e.message}"
       return head :bad_request
     end
+
     # Check if webhook signing is configured.
     if AppSecrets.STRIPE_WEBHOOK_SECRET.present?
       # Retrieve the event by verifying the signature using the raw body and secret.
       signature = request.env['HTTP_STRIPE_SIGNATURE']
+
```

Comment:
I put these line breaks in kind've instinctively as they seem more readable, can remove